.github/workflows/tests.yaml

```diff
@@ -28,12 +28,34 @@ jobs:
 
       # TODO: Cache yarn install
 
-      - name: "test"
+      - name: Set variables
+        id: variables
+        working-directory: packages/${{ matrix.package }}
         run: |
-          cd packages/${{ matrix.package }}
-          yarn install
-          yarn tsc
-          HAS_TESTS=$(cat package.json | jq '.["scripts"]["test"]')
-          if [[ $HAS_TESTS != "null" ]]; then
-            yarn test
-          fi
+          echo "DM_CORE_VERSION=$(cat package.json | jq '.["dependencies"]["@development-framework/dm-core"]')" >> "$GITHUB_OUTPUT"
+          echo "TEST_COMMAND=$(cat package.json | jq '.["scripts"]["test"]')" >> "$GITHUB_OUTPUT"
+
+      - name: 'Install dependencies'
+        working-directory: packages/${{ matrix.package }}
+        run: yarn install
+
+      - name: 'Copy local variant of dm-core to node_modules'
+        working-directory: packages/${{ matrix.package }}
+        if: steps.variables.outputs.DM_CORE_VERSION != 'null'
+        run: |
+          cd ../dm-core
+          yarn install --no-lockfile
```

Comment:
yarn install is already done at line 40.

Reply:
Yes, but that installs the dependencies in dm-core-plugins. This one installs it in dm-core. We need both